### ISSUE TITLE
gtk_tree_view_get_model not exported anymore

### ISSUE THREAD

**Issue by Dankr4d** [StefanSalewski/gintro#57]
The gtk_tree_view_get_model proc is not exported anymore and the first listview example will not work/compile.
```
Error: undeclared identifier: 'gtk_tree_view_get_model'
```

**Comment by StefanSalewski** [comment]
You are right, the low level procs starting with prefixes like gtk_ are not exported. They where exported only in early versions of gintro, but they where never considered to be really used in high level code.  The expected Nim  high level name seems to be getModel(), maybe plain model(). For first example you may be reffering to listview.nim? Compiles and works fine here.

```
$ grep gtk_tree_view_get_model gintro/examples/gtk3/*
stefan@nuc ~ $ grep getModel gintro/examples/gtk3/*
gintro/examples/gtk3/css_colored_listview.nim:  let store = listStore(tree.getModel())   
gintro/examples/gtk3/listview.nim:  let store = listStore(getModel(list))
gintro/examples/gtk3/listview.nim:  let store = list.getModel.listStore#getListStore(list)
gintro/examples/gtk3/listview.nim:  let store = list.getModel.listStore
```

Maybe the example code in the html github page does not compile -- well I told you to use the examples from examples directory, the code in html page is manually inserted years ago, github does to allow inserting code from files directly. I do intent creating a new tutorial with directly inserted code hosted somewhere else.

**Comment by Dankr4d** [comment]
I'll give it a try later and test your new examples from within the example folder. Then I'll rewrite my current listview in my project, that currently builds upon the old example of your readme. So I'll give you response later. Thank you for your reply. I thought so that the low level functions will not exported anymore at some state of this project. Opened this issue to prevent some confusion for new users of this project.

**Comment by Dankr4d** [comment]
Oh, I forgot this issue. As you mentioned I fixed the listview.
From:
```nim
proc getListStore(self: TreeView): ListStore =
  new(result)
  result.impl = gtk_tree_view_get_model(cast[ptr TreeView00](self.impl))
```
To (from: https://github.com/StefanSalewski/gintro/blob/master/examples/gtk3/listview.nim):
```nim
proc typeTest(o: gobject.Object; s: string): bool =
  let gt = g_type_from_name(s)
  return g_type_check_instance_is_a(cast[ptr TypeInstance00](o.impl), gt).toBool

proc listStore(o: gobject.Object): gtk.ListStore =
  assert(typeTest(o, "GtkListStore"))
  cast[gtk.ListStore](o)
```
Should I close this issue?

**Comment by StefanSalewski** [comment]
I think we can leave this issue open for now, it may help some other people still. For the future we should try to further simplify listview use and write a longer listview tutorial, as listview is one of the most difficult parts of gtk. I read about list- and treeview in the Krause book 10 years ago, but can not remember details. I would have to read it again before writing about listview.

Currently my next goal is to get the new --gc:ARC working with gintro...
